certora/harness/MorphoHarness.sol

```diff
@@ -22,6 +22,14 @@ contract MorphoHarness is Morpho {
         return idToMarketParams[id];
     }
 
```

Comment:
we could have a little comment, as it's not trivial why one would need this

Reply:
Also, this function is not from this PR. What shall the comment say?

Reply:
I meant the function that you changed